README.md

```diff
@@ -72,13 +72,43 @@ RNA-seq limma+voom:
      -profile rnaseq_limma,<docker/singularity/podman/shifter/charliecloud/conda/institute>
 ```
 
-:::note
-If you are using the outputs of the nf-core rnaseq workflow as input here you should provide either the **gene_counts_length_scaled.tsv** or **gene_counts_scaled.tsv** matrices. This follows the [recommendation from the tximport documentation](https://bioconductor.org/packages/devel/bioc/vignettes/tximport/inst/doc/tximport.html#limma-voom):
+> [!NOTE]
+> To use advanced mixed model in your analysis, you need to prepare your submission sheet by combining experimental factors into a single composite variable. See the necessary steps:
+
+##### Step 1: Create a Composite Factor Column
```

Comment:
Actually, this material is a bit involved to go in the README, that's really just an overview, could we have a new section in the usage docs about mixed models please? Maybe just before the Gene Set bit? Open to suggestions though https://github.com/nf-core/differentialabundance/blob/3dd360fed0dca1780db1bdf5dce85e5258fa2253/docs/usage.md?plain=1#L276

Reply:
I agree mixed models description was moved to usage section